test/CMakeLists.txt

```diff
@@ -17,4 +17,4 @@ add_subdirectory(old-tests)
 
 add_subdirectory(install)
 
-configure_file(${CMAKE_SOURCE_DIR}/test/CTestCustom.cmake ${CMAKE_BINARY_DIR})
+configure_file(${CMAKE_CURRENT_LIST_DIR}/CTestCustom.cmake ${CMAKE_CURRENT_BINARY_DIR})
```

Comment:
CMAKE_CURRENT_LIST_DIR is the fix. I changed CMAKE_BINARY_DIR to CMAKE_CURRENT_BINARY_DIR because the latter is used almost everywhere else in the test directory.